index.js

```diff
@@ -70,6 +79,11 @@ module.exports = function newmanCSVReporter (newman, options) {
 
     log[key] = log[key] || []
     log[key].push(assertion)
+
+    log['totalAssertions']++
+    const assertionKey = (key == 'executed') ? 'passedCount' :
+                        (key == 'skipped') ? 'skippedCount' : 'failedCount'
+    log[assertionKey]++
```

Comment:
Could this just be simplified to:

```
log[`${key}Count`]++
log['totalAssertions']++
```

where line 17 (also line 60 would need a similar update) would also change to:

````
'executedCount',
```

I understand using `passed` but the reporter already uses `executed` which is consistent with bundled reporters in Newman such as the default CLI reporter.